spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/NacosConfigProperties.java

```diff
@@ -89,6 +89,11 @@ public class NacosConfigProperties {
 	 */
 	public static final String DEFAULT_NAMESPACE = "public";
 
+    /**
+     * Nacos default server and port.
+     */
```

Comment:
Code format needs to be adjusted

Reply:
Is there any documentation on what the developers can do to meet the project styles?